app/schedulers/remove_unverified_user_cron_job.py

```diff
@@ -0,0 +1,22 @@
+import time
+from app.api.email_utils import EMAIL_VERIFICATION_TOKEN_TIME_TO_EXPIRE
+
+def remove_unverified_user_job():
+    """
+    This function iterates over all the users and
+    checks if a user has unverified email and
+    the verification token of the user has expired.
+    if True then the user is removed, if False does nothing.
+    """
+    from run import application
+    with application.app_context():
+        from app.database.models.user import UserModel
+        all_users = UserModel.query.all()
```

Comment:
filter by unverified please

Reply:
But doing so will decrease the efficiency of the code. If the filter function is applied over here, then internally a loop would run to filter the list, and then another loop will run to remove the unverified users. Currently this work is done only by a single loop.